browser/services/platform.js

```diff
@@ -50,6 +50,16 @@ class Platform {
     });
   }
 
+  static getProgramFilePath() {
+    return Platform.identify({
+      win32: function() {
+        return pify(child_process.exec)('echo %PROGRAMFILES%').then((stdout)=>{
```

Comment:
Yo can get env variable value just from `process.env.PROGRAMFILES`, no need to run async command.